locale/en/blog/release/v5.0.0.md

```diff
@@ -0,0 +1,202 @@
+---
+date: 2015-10-29T12:33:38.068Z
+version: 5.0.0
+category: release
+title: Node v5.0.0 (Stable)
+slug: node-v5-0-0
+layout: blog-post.hbs
+---
```

Comment:
Hmm, `author` field not added, did you use an updated version of `./scripts/release-post.js`?